packages/scrum-client/src/content/Signup/Signup.js

```diff
@@ -36,24 +41,31 @@ export default function Signup() {
       confirmPassword.length > 0
     );
   }
+  const isValidForm = validateForm();
+  const isReferral = checkReferral(referrer);
 
   function handleSubmit(event) {
     event.preventDefault();
 
     if (password !== confirmPassword) {
       setErrorMessage("Passwords don't match");
     } else {
+      const payload = {
```

Comment:
Any better way to do this?

Reply:
Have you try  `{...payload, ...{referredBy: referrer}}}`
Undefined values wont be assigned by the spread operator

Reply:
A veces el referredBy es un texto, por el menu que tenemos, entonces lo dejare como esta